### PR TITLE
draft whats new

### PR DESCRIPTION
**Pull request by marqh**
draft what's new page

clean up of contributions files

### CODE REVIEW

File: docs/iris/src/whatsnew/1.13.rst [hunk 1]
```diff
@@ -0,0 +1,37 @@
+What's New in Iris 1.12
```
Comment (QuLogic) [inline docs/iris/src/whatsnew/1.13.rst]: 1.13

File: docs/iris/src/whatsnew/1.13.rst [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+***********************
+
+:Release: 1.12
```
Comment (QuLogic) [inline docs/iris/src/whatsnew/1.13.rst]: 1.13

File: docs/iris/src/whatsnew/1.13.rst [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+:Date: 2017-05-17
+
+
+This document explains the new/changed features of Iris in version 1.12
```
Comment (QuLogic) [inline docs/iris/src/whatsnew/1.13.rst]: 1.13

File: docs/iris/src/whatsnew/1.13.rst [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+* Allow the reading of NAME trajectories stored by time instead of by particle number.
+* An experimental link to python-stratify via :mod:`iris.experimental.stratify`
+* Data arrays may be shared between cubes, and subsets of cubes, by using the :meth:`iris.cube.share_data` flag.
+
+
+Bug Fixes
+=========
+
+* The bounds are now set correctly on the longitude coordinate if a zonal mean diagnostic has been loaded from a PP file as per the CF Standard.
+* netCDF loading will now determine whether there is a string-valued scalar label, i.e.: a character variable that only has one dimension (the length of the string) and interpret this effectively.
```
Comment (QuLogic) [inline docs/iris/src/whatsnew/1.13.rst]: NetCDF

File: docs/iris/src/whatsnew/1.13.rst [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+* A line plot of geographic coordinates (e.g. drawing a trajectory) wraps around the edge of the map cleanly, rather than plotting a segment straight across the map.
+* Preserve lazy data in PP save-pairs generation, so you can work with a list of save cubes and/or fields without excessive memory requirements (as intended for saving).
+* When two cubes have coordinates which do not match, an error should be raised if a user tries to perform arithmetic operations on them. These cases are now caught by the multiply and divide operators as well as the add and subtract operators, which already worked effectively.
+* Limited area Rotated Pole datasets where the data range is 0 <= lambda < 360, for example as produced in New Zealand, are plotted over a sensible map extent by default.
+* Removed  the potential for a RuntimeWarning: overflow encountered in int_scalars which was missed during collapsed calculations.  This could can trip up unwary users of limited data types, such as int32 for very large numbers (e.g. seconds since 1970).
```
Comment (QuLogic) [inline docs/iris/src/whatsnew/1.13.rst]: could can?

File: docs/iris/src/whatsnew/1.13.rst [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+* The CF conventions state that certain formula_terms terms may be omitted and assumed to be zero (http://cfconventions.org/cf-conventions/v1.6.0/cf-conventions.html#dimensionless-v-coord) so Iris now allows factories to be constructed with missing terms.
+* clabel inline is set to be False for line countour plots. Clabel does not remove any of the contour so it renders correctly, avoiding spurious horizontal lines across plots, although this does make labels a little harder to see
```
Comment (QuLogic) [inline docs/iris/src/whatsnew/1.13.rst]: countour -> contour.

Also, I would not put "Clabel/clabel" at the beginning of a sentence as it doesn't capitalize well and it looks weird.

**Comment by lbdreyer** [comment]
As @marqh is unavailable to finish off this PR, I have taken his commit and made some edits (including from @QuLogic 's review) and opened #2561

**Pull request closed without merging**